index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+<body>  
+    <div class="screen">
+        <header class="header">
+            <div class="time-status">
+                <div class="time">04:20 am</div>
+                <div class="status">
+                   <img src="images/status.svg" alt="status" class="status"> 
+                </div>
+            </div>
+            <div class="header-items">
```

Comment:
![white-space (copy)](https://user-images.githubusercontent.com/93039359/185613123-68eb37d1-e5da-45bd-90aa-092032b543f0.png)


- [ ] Please add the following code at the very start of your style.css file to remove the white space on top and left :+1:

```
body {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```